### PR TITLE
fix `torch.futures` docstring examples

### PR DESCRIPTION
**Pull request by pmeier**
Trying to run the doctests for the complete documentation hangs if it reaches the examples of `torch.futures`. It turns out to be only syntax errors, which are normally just reported. My guess is that `doctest` probably doesn't work well for failures within async stuff. 

Anyway, while debugging this, I fixed the syntax.

### CODE REVIEW

**Comment by mruberry** [comment]
@mrshenli -- you OK to review this?

File: torch/futures/__init__.py [hunk 1]
```diff
@@ -200,19 +195,14 @@ def add_done_callback(self, callback):  # type: (Callable[[Future[T]], None]) ->
             for handling completion/waiting on those futures independently.
 
         Example::
-            >>> import torch
-            >>>
             >>> def callback(fut):
-            >>>     print(f"This will run after the future has finished.")
-            >>>     print(fut.wait())
-            >>>
+            ...    print(f"This will run after the future has finished.")
```
Comment (mrshenli) [inline torch/futures/__init__.py:199]: missing one space?

**Review by mrshenli — APPROVED** [review]
Doc fixes LGTM. Thanks for fixing!

Did you get a chance to check that the result renders correctly in our doc page?

File: torch/futures/__init__.py (hunk 1 above)
Comment (pmeier) [reply torch/futures/__init__.py]: Good catch!

**Comment by pmeier** [comment]
@mrshenli 

> Did you get a chance to check that the result renders correctly in our doc page?

You can find the rendered version [here](https://14631391-65600975-gh.circle-artifacts.com/0/docs/futures.html).

**Pull request closed without merging**